### PR TITLE
chore(profile-creation): Remove all reliance on pb consts

### PR DESCRIPTION
**Pull request by ChefHutch**
Depends on https://github.com/pancakeswap/pancake-frontend/pull/2419

`src/config/constants/nfts/pancakeBunnies.ts` ded 🥳

### CODE REVIEW

File: src/config/constants/nftsCollections/types.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Address } from '../types'
+
+export enum CollectionKey {
+  PANCAKE = 'pancake',
+  SQUAD = 'pancakeSquad',
+}
+
+export type Collection = {
```
Comment (ChefHutch) [inline src/config/constants/nftsCollections/types.ts:8]: The Collection constants live to fight another day, for now (we use `pancakeBunniesAddress` a _lot_ and there's no need to remove our reliance on these until we're whitelisting non-PCS nft projects, or event beyond that).

The individual NFTs are all ded.

File: src/views/ProfileCreation/config.ts [hunk 2]
```diff
@@ -3,6 +3,6 @@ import { parseUnits } from '@ethersproject/units'
 export const MINT_COST = parseUnits('1')
 export const REGISTER_COST = parseUnits('0.5')
 export const ALLOWANCE_MULTIPLIER = 5
-export const STARTER_BUNNY_IDENTIFIERS = ['sleepy', 'dollop', 'twinkle', 'churro', 'sunny']
+export const STARTER_NFT_BUNNY_IDS = ['5', '6', '7', '8', '9']
```
Comment (0xCorgi) [inline src/views/ProfileCreation/config.ts:6]: Should use constants here to remember what those ids are?

File: src/views/ProfileCreation/Mint.tsx [hunk 3]
```diff
@@ -37,6 +40,19 @@ const Mint: React.FC = () => {
   const hasMinimumCakeRequired = fetchStatus === FetchStatus.SUCCESS && cakeBalance.gte(MINT_COST)
   const { callWithGasPrice } = useCallWithGasPrice()
 
+  useEffect(() => {
+    const getStarterNfts = async () => {
+      const { data: allPbTokens } = await getNftsFromCollectionApi(pancakeBunniesAddress)
+      const nfts = STARTER_NFT_BUNNY_IDS.map((bunnyId) => {
+        return { ...allPbTokens[bunnyId], bunnyId }
```
Comment (0xCorgi) [inline src/views/ProfileCreation/Mint.tsx:47]: Should guard if `allPbTokens[bunnyId]` is not defined or not necessary?

**Review by 0xCorgi — APPROVED** [review]
Looks good ! 
Just a few comments.

File: src/views/ProfileCreation/config.ts (hunk 2 above)
Comment (hachiojidev) [reply src/views/ProfileCreation/config.ts:6]: Yeah, agreed. Or at the very least we should comment what each bunny id is.

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Nice. Don't forget to remove the test.

**Comment by hachiojidev** [comment]
@ChefHutch Rebase.

**Comment by ChefHutch** [comment]
> @ChefHutch Rebase.

This was always branching off something on my fork - and I just changed its base to _my_ fork's develop. Carnage. Remaking the PR.

**Comment by ChefHutch** [comment]
Correct base (`pancakeswap:develop`): https://github.com/ChefHutch/pancake-frontend/pull/32

**Pull request closed without merging**